core/src/test/java/com/linecorp/armeria/server/file/HttpFileServiceTest.java

```diff
@@ -324,6 +325,26 @@ public void testFileSystemGet() throws Exception {
         }
     }
 
+    @Test
+    public void testFileSystemGet_newFile() throws Exception {
```

Comment:
It does not test the case of adding a new file but modifying an existing file. Could you:

1. Rename this method to `testFileSystemGet_modifiedFile` and
2. Add another test method `testFileSystemGet_newFile`, which makes sure the first request returns 404 and the second request returns 200

..?

Reply:
Thank you! I will add another test for newFile case,
I would like to confirm:
From the unit test, my understand is `final File barFile = new File(tmpDir, "bar.html");` this is like the place that saving cached file, so if we return new html, the content in this path will be refreshed, am I right?

Reply:
Yes. To be precise, the cached content will be refreshed.